[Bug]: build and test documentation require a new version of Quarto

### Describe the bug


When building locally, I encounter this error when running `python process_notebooks.py render`:

```
...
Quarto version is too old. Please upgrade to 1.5.23 or later.
```

And when run `yarn start` it fails.
```
[ERROR] Sidebars file at "/Users/kevin/Desktop/Desktop files/playground/autogen/website/sidebars.js" failed to be loaded.
[ERROR] Loading of version failed for version current

Error: Sidebar category Topics has neither any subitem nor a link. This makes this item not able to link to anything.
```

I am using MacOS 13.4 (22F66) (Apple Chip), and I installed Quarto=1.4.551 from the link in contributing document.
The problem is solved when installing a prelease version quarto=1.5.24 from https://quarto.org/docs/download. 

@sonichi @skzhang1 @qingyun-wu Should I make a PR to update the version? I am not sure this happens to other OSs.



### Steps to reproduce

_No response_

### Model Used

_No response_

### Expected Behavior

_No response_

### Screenshots and logs

_No response_

### Additional Information

_No response_

--- a/website/docs/Contribute.md
+++ b/website/docs/Contribute.md
@@ -203,22 +203,23 @@ Then you can see the coverage report by
 `coverage report -m` or `coverage html`.
 
 ### Documentation
-
-To build and test documentation locally, install [Node.js](https://nodejs.org/en/download/). For example,
-
+#### Build documentation locally
+1\. To build and test documentation locally, first install [Node.js](https://nodejs.org/en/download/). For example,
 ```bash
 nvm install --lts
 ```
 
-Also, install [quarto](https://quarto.org/docs/get-started/).
+Then, install `yarn` and other required packages:
+```bash
+npm install --global yarn
+pip install pydoc-markdown pyyaml termcolor
+```
 
-> Note: Support for Docusaurus 3.0 in Quarto is from version `1.4`. Ensure that your `quarto` version is `1.4` or higher.
+2\. You also need to install quarto. Please click on the `Pre-release` tab from [this website](https://quarto.org/docs/download/) to download the latest version of `quarto` and install it. Ensure that the `quarto` version is `1.5.23` or higher.
 
-Then:
+3\. Finally, run the following commands to build:
 
 ```console
-npm install --global yarn  # skip if you use the dev container we provided
-pip install pydoc-markdown pyyaml termcolor # skip if you use the dev container we provided
 cd website
 yarn install --frozen-lockfile --ignore-engines
 pydoc-markdown
@@ -229,7 +230,8 @@ yarn start
 The last command starts a local development server and opens up a browser window.
 Most changes are reflected live without having to restart the server.
 
-To build and test documentation within a docker container. Use the Dockerfile in the `dev` folder as described above to build your image
+#### Build with Docker
+To build and test documentation within a docker container. Use the Dockerfile in the `dev` folder as described above to build your image:
 
 ```bash
 docker build -f .devcontainer/dev/Dockerfile -t autogen_dev_img https://github.com/microsoft/autogen.git#main

